ninejs/static/default.css

```
@@ -1,46 +1,48 @@
-}
+  .tooltip {
+    position: absolute;
+    background: #001d3d;
+    padding: 8px 12px;
+    border-radius: 6px;
+    color: #ffffff;
+    font-size: 14px;
+    box-shadow: 0 4px 12px rgba(0, 0, 0, 0.1);
+    pointer-events: none;
+    display: none;
+    font-family: "Helvetica Neue", "Arial", sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Arial" (font-family-name-quotes)

(font-family-name-quotes)